### PR TITLE
Minor dev tooling improvements

### PR DESCRIPTION
**Pull request by Stringy**
Hi folks, I've just been fiddling around a little (hope to contribute more substantial things in the future) but here's some little improvements to the build tooling that I found helpful when I was getting up and running today:

- uses platform defaults for data path (can still be overridden in foundryconfig.json)
- updates README with some basic commands for getting started

I'm running/developing on Linux, hence the additional platform defaults alongside Windows.

### CODE REVIEW

**Review by mkscho63 — APPROVED** [review]
Love it, thanks for making this change

File: README.md
Comment (WesWedding) [inline README.md:1]: Good changes, anything to make it a little easier for folks to contrib.

File: gulpfile.js
Comment (WesWedding) [inline gulpfile.js:1]: This worked locally on my Win11 machine.

**Review by WesWedding — APPROVED** [review]
Looks like a good update, and tested well on my local setup.

**Pull request merged**